js/util.ts

```diff
@@ -207,3 +208,90 @@ export function splitNumberToParts(n: number): number[] {
   const higher = (n - lower) / 0x100000000;
   return [lower, higher];
 }
+
+// Utility functions for DOM nodes
```

Comment:
I think the error is caused by making changes to js/util.ts ... Note the comment 
```
// Because this is accessing the live bundle, this test might be more fragile 
```
Furthermore, I don't think these belong here anyway. `js/util.ts` is more for `assert()` and other things that really don't have many dependencies - by adding the DOM stuff here you're creating a lot of dependencies to DOM in other modules that have nothing to do with it.
I suggest moving these into `js/event_target.ts` since it seems like they're not used elsewhere. If you really need a utility module, create something like `js/event_util.ts`.